packages/shared-ux/page/solution_nav/src/with_solution_nav.tsx

```diff
@@ -18,31 +18,30 @@ function getDisplayName(Component: ComponentType<any>) {
   return Component.displayName || Component.name || 'UnnamedComponent';
 }
 
-type TemplateProps = Pick<
-  EuiPageTemplateProps,
-  'pageSideBar' | 'pageSideBarProps' | 'template' | 'children'
->;
-
-type ComponentProps = TemplateProps & {
-  isEmptyState?: boolean;
-};
+// TODO: Would be nice to grab these from KibanaPageTemplate or vice-versa
```

Comment:
If this is a follow-up for Shared UX team, we should file an issue and ref it here.  Perhaps a meta "follow up" issue to track them all, along with some details?

Reply:
Hey hey! For context, Caroline added these TODOs when she was first starting up the PR, so I'm much less familiar with their context/intention 🙈 Apologies for that - but yes, I'd be happy to file a follow-up issue from this PR for the Shared UX team. Thanks so much!